service/implementations/person_implementation.rb

```diff
@@ -0,0 +1,93 @@
+class PersonImplementation < Hoodoo::Services::Implementation
+
+  # def show( context )
+  #   person = Person.acquire_in!( context )
+  #   return if context.response.halt_processing?
+  #   context.response.set_resources( render_in( context, person ) )
+  # end
+  def show( context )
+    person = Person.acquire_in( context )
+    if person.nil?
+      context.response.not_found( context.request.ident )
+    else
+      context.response.set_resource( render_in( context, person ) )
+    end
+  end
+
+  def list( context )
+    # context.request.list.limitv= 1000 if context.request.list.limit > 1000 # Limit the size of page size a caller can request.
```

Comment:
Can you guess why this might be useful to be kept in?
```suggestion
    context.request.list.limit= 1000 if context.request.list.limit > 1000 # Limit the size of page size a caller can request.
```